packages/gestalt/src/TableRowDrawer.js

```diff
@@ -52,11 +52,13 @@ export default function TableRowDrawer({ children, drawerContents, id }: Props):
       <tr aria-details={id} ref={rowRef}>
```

Comment:
This id should be left off if there are no drawerContents anymore

Reply:
Good a11y catch

Reply:
DONE